### ISSUE TITLE
[Bug][a11y] Insufficient contrast of the orange spinner for button in loading state

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#437]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

iOS 18.3

### App version

0.10.0

### App build number

1737966409

### App build type

alpha

### Describe the issue

On component BUTTON, in loading state, the contrast of the orange (#f16e00) spinner is not sufficient with light background (white or light gray) for mobile use.

![Image](https://github.com/user-attachments/assets/836dd3c4-28f3-474b-8a59-ed866031f5f4)

### Expected Behavior

Mobile recommendation is to reach the WCAG level AAA for color contrast, meaning : 

- 4.5:1 for large texts or non-text contents,
- 7:1 for texts.

Better to use black color instead for the loader

### Accessibility settings

"High contrast" mode doesn't have effect on the component itself.
You could check if the user has enabled this mode thanks to @Environment(\.colorSchemeContrast) and increase the contrast of the component

**Comment by pya35** [comment]
@B3nz01d @MaximeTonnerre #f16e00 or #f15e00 is not sufficient for components on mobile apps. 
[4.5:1 ratio is expected for mobile usage](https://system.design.orange.com/0c1af118d/p/077e99-accessibility-compliance)(WCAG level AAA).

It'd be interesting to think about new accessible color palette for this new design system

**Comment by B3nz01d** [comment]
Completed during Sprint 19

**Comment by pya35** [comment]
OK in high-contrast mode, the spinner turns black in light mode
